assembly/__tests__/env-coins.spec.ts

```diff
@@ -52,6 +60,18 @@ describe('Testing env coins related functions', () => {
     expect(env.balanceOf(testAddress2.toString())).toBe(amount);
   });
 
+  it('transferCoins to mock address created with addAddressToLedger', () => {
```

Comment:
i find your test description quite hard to understand :/
It seems your test is just doing a simple transfercoins ? isn't it?

Reply:
Yes I'm not sure it's usefull now because we should not be able to create a new address with undifined balance